blueprints/layout/files/__root__/layouts/__name__Layout/index.js

```diff
@@ -1,4 +1,5 @@
 const React = require('react');
+const T = require('prop-types');
```

Comment:
While this is a change I think we should make, I don't know if it belongs in the login recipe.

Reply:
want me to make a separate pr or just leave it alone? I need to update the readme too, so i can do it around the same time

Reply:
Yeah, my suggestion would be to remove these changes from this PR, and possibly make a separate PR altogether.